source/importers/1PasswordImporter.js

```diff
@@ -1,9 +1,93 @@
-(function(module) {
+"use strict";
 
-	"use strict";
+const path = require("path");
+const fs = require("fs");
 
-	var Importer = function(filename, destination) {
+const pify = require("pify");
+const isDir = require("is-dir").sync;
+const Buttercup = require("buttercup");
 
-	};
+const {
+	convert1pifToJSON
+} = require("../tools/1password.js");
 
-})(module);
+const {
+	Archive
+} = Buttercup;
+const readFile = pify(fs.readFile);
```

Comment:
[`fs-extra`](https://github.com/jprichardson/node-fs-extra) returns Promises too. but this is good as well. just saying